thrimbletrimmer/edit.html

```diff
@@ -205,6 +205,11 @@ <h2>Keyboard Shortcuts</h2>
 					<input type="checkbox" id="advanced-submission-option-allow-holes" />
 				</div>
 
+				<div>
+					<label for="advanced-submission-option-public">Make public (not unlisted)</label>
+					<input type="checkbox" id="advanced-submission-option-public" />
+				</div>
+
```

Comment:
Might be better to have the logic reversed here? As in, a default-unchecked checkbox for "Unlisted".